### PR TITLE
Created a new function checkingDuplicateMessage for optimising

### PR DESCRIPTION
**Pull request by abhisheksingh-jsdev**
Created a new function checkingDuplicateMessage for optimising MessagInCID. This function checks in for duplicateMessage and returns array of Messages

### CODE REVIEW

**Review by corlard3y — CHANGES_REQUESTED** [review]
In the Chats section of this branch, after sending messages - the previous messages dissappear on the UI and I cannot see them. This happens after sending new messages. @abhisheksingh-jsdev

File: src/components/chat/w2wChat/chatBox/ChatBox.tsx [hunk 1]
```diff
@@ -116,6 +116,31 @@ const ChatBox = ({ setVideoCallInfo }): JSX.Element => {
     }
   }, [currentChat]);
 
+  const getMessageIndex = ({newMessages,msgIPFS} : {newMessages:MessageIPFSWithCID[], msgIPFS:MessageIPFSWithCID}): Number=>{
```
Comment (mishramonalisha76) [inline src/components/chat/w2wChat/chatBox/ChatBox.tsx:119]: shift these to helper folder

File: src/components/chat/w2wChat/chatBox/ChatBox.tsx [hunk 2]
```diff
@@ -116,6 +116,31 @@ const ChatBox = ({ setVideoCallInfo }): JSX.Element => {
...
+        msg.encType === '' &&
+        msg.cid === '' &&
+        msg.messageContent === msgIPFS.messageContent &&
+        msg.messageType === msgIPFS.messageType
+    );
+    console.log("Index",index)
+    return index
+  }
+
+  const getDuplicateMessage = ({messages,msgIPFS} : {messages:MessageIPFSWithCID[], msgIPFS:MessageIPFSWithCID}) : MessageIPFS=>{
```
Comment (mishramonalisha76) [inline src/components/chat/w2wChat/chatBox/ChatBox.tsx:132]: can create a common param type for both

File: src/components/chat/w2wChat/chatBox/ChatBox.tsx (hunk 1 above)
Comment (mishramonalisha76) [inline src/components/chat/w2wChat/chatBox/ChatBox.tsx:119]: Number type should be number

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**